index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fsCallback = require("./lib/files-callBack.js");
+const fsPromise = require("./lib/file-promise.js"), write = require("./lib/write");
+const file = process.argv.slice(2)[0];
+
+
+
+/** this is just a function to run 
+ * callBack with two parameters */
+
+function myTestcallBack(err, data) {
+  console.log("hi!")
+}
```

Comment:
Test functions that don't serve a purpose for the lab should be removed when you submit.